src/help/doc/audit.md

```diff
@@ -5,6 +5,6 @@ Setup a custom IAM Role per Lambda function
 ## Example Usage
 
 -  `npx audit` displays all Lambda functions and associated IAM Role
--  `npx config apply` sync policies found in Lambda function role.json to unique IAM Role
+-  `npx config {apply | --apply | -a}` sync policies found in Lambda function role.json to unique IAM Role
```

Comment:
convention here would be [] brackets for option arguments ala `npx config [apply|--apply|-a]`

Reply:
Switched over to []